templates/includes/sidebar.html

```diff
@@ -1,6 +1,19 @@
-<ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
-</ul>
+<div class="sidebar">
+    {% if user.is_authenticated %}
+        <p>Welcome, 
+            <a href="{% url 'taxi:driver-detail' user.id %}">
```

Comment:
Ensure that the `user.id` corresponds to a driver. If the user model is not directly linked to the `Driver` model, this URL might not work as expected. Consider adding a check or modifying the logic to ensure the correct ID is used.